types/part_set.go

```diff
@@ -170,7 +170,7 @@ type PartSet struct {
 // The data bytes are split into "partSize" chunks, and merkle tree computed.
 // CONTRACT: partSize is greater than zero.
 func NewPartSetFromData(data []byte, partSize uint32) *PartSet {
-	// divide data into 4kb parts.
+	// divide data into 64kB parts (default size of one block part is 64kB).
```

Comment:
```suggestion
	// divide data into parts of size `partSize`
```

Reply:
Suggesting this as the unit tests use non-default values quite often (so the default size depends on the callsite)